install/kubernetes/README.md

```diff
@@ -1,13 +1,15 @@
 # Install Istio on an existing Kubernetes cluster
 
-Please follow the installation instructions from [istio.io](https://istio.io/docs/tasks/installing-istio.html).
+Please follow the installation instructions from [istio.io](https://istio.io/docs/setup/kubernetes/quick-start.html).
 
 ## Directory structure
 This directory contains files needed for installing Istio on a Kubernetes cluster.
 
 * istio.yaml - use this file for installation without authentication enabled
 * istio-auth.yaml - use this file for installation with authentication enabled
-* istio-cluster-wide.yaml - use this file for installation cluster-wide with authentication enabled
+* istio-initializer.yaml - use this file for installation of istio initializer for transparent injection.
+* istio-one-namespace.yaml - use this file for installation without authentication enabled and CA. Istio control plane and applications will be in one single namespace, mainly used for testing.
+* istio-one-namespace-auth.yaml - use this file for installation without authentication enabled and CA. Istio control plane and applications will be in one single namespace, mainly used for testing.
```

Comment:
replace without with with.

Reply:
remove CA reference from both. CA is always installed.

Reply:
Thanks @andraxylia , it is a bit late to update as it was already merged, created a new PR here https://github.com/istio/istio/pull/960